roles/shell/tasks/oh-my-zsh.yml

```diff
@@ -12,16 +12,16 @@
 
 - name: Fetch .zshrc from remote machine
   fetch:
-    src: /root/.zshrc
+    src: "{{ '/home/' + user + '/.zshrc' if user != 'root' else '/root/.zshrc' }}"
```

Comment:
can we handle it using `ansible_user` instead of set condition for it? How does the user is set? does it come from the `ansible_user`?

Reply:
Yes, variable `{{ user }}` comes from `ansible_user` which resides in `/roles/shell/vars/main.yml`.
Can we just use `anisble_user` for less complexity?